packages/matchbox/src/components/Toggle/styles.js

```diff
@@ -0,0 +1,68 @@
+import { tokens } from '@sparkpost/design-tokens';
+import styled from 'styled-components';
+
+export const toggle = props => `
+  display: inline-block;
+  position: relative;
+  height: 1.25rem;
+  width: 2.25rem;
```

Comment:
You probably already thought of this, but would it be possible to leverage design tokens here? May not actually work well visually, so totally fine if not.

Reply:
Yeah, I'm realizing we have a gap in our spacing tokens. They were really only meant to handle margin and padding, not things like size. But either way, I think this is fine as long as we are using rems and referencing a baseline font size token (which we have a ticket for)